src/lib/systemjs-angular-loader.js

```diff
@@ -0,0 +1,49 @@
+var templateUrlRegex = /templateUrl\s*:(\s*['"`](.*?)['"`]\s*)/gm;
```

Comment:
Can't it be inlined in the karma configuration like the rest of the current unit testing configs?